src/core/connection.h

```diff
@@ -630,6 +630,11 @@ typedef struct QUIC_CONNECTION {
     //
     uint16_t KeepAlivePadding;
 
+    //
+    // Indicates that the peer accepts RELIABLE_RESET kind of frames, in addition to RESET_STREAM frames.
+    //
+    uint8_t PeerReliableResetEnabled;
```

Comment:
Rename this to something like `ReliableResetStreamNegotiated` and only set it to TRUE if the local app enables it and the remote/peer does too.

Reply:
I am guessing once we add the new connection event, that's where we will set this flag?